modules/text_server_adv/text_server_adv.cpp

```diff
@@ -7229,6 +7229,17 @@ bool TextServerAdvanced::_is_valid_identifier(const String &p_string) const {
 	return true;
 }
 
+bool TextServerAdvanced::_is_valid_letter(char32_t p_unicode) const {
+#ifndef ICU_STATIC_DATA
+	if (!icu_data_loaded) {
+		WARN_PRINT_ONCE("ICU data is not loaded, Unicode security and spoofing detection disabled.");
```

Comment:
This is not related to spoofing and security, not sure if any warning is necessary.

Reply:
Sure, I can remove it. Copied it from `TextServerAdvanced::_is_valid_identifier` just in case